src/main/cpp/avif_codec.h

```diff
@@ -0,0 +1,42 @@
+#ifndef LIBENCH_AVIF_H
+#define LIBENCH_AVIF_H
+
+#include "codec.h"
+
+#include "avif/avif.h"
+
+namespace libench {
+
+class AVIFEncoder : public Encoder {
+ public:
+  AVIFEncoder();
+  ~AVIFEncoder() override;
```

Comment:
I used the C++11 new keyword `override` when overriding a virtual method of the base class. Hope this is okay.